app/src/main/java/com/kylecombes/micrepeater/ui/main/VoiceAmplifierFragment.java

```diff
@@ -37,6 +46,18 @@ static VoiceAmplifierFragment newInstance() {
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         pageViewModel = new ViewModelProvider(Objects.requireNonNull(getActivity())).get(AppStateViewModel.class);
+        String output = getActivity().getSharedPreferences("PREFERENCE", Context.MODE_PRIVATE)
+                .getString("audioOutput", "voice");
+        Log.v("out", output);
```

Comment:
Maybe log a meaningful message here instead of just `out`

Reply:
The other activities and fragments typically use their class name as the tag:
```java
    private static final String TAG = VoiceAmplifierFragment.class.getCanonicalName();
```
So
```java
Log.v(TAG, output);
```
Also, if we don't need this for debugging, we could just delete this line.